boards/arm/cc3220sf_launchxl/doc/cc3220sf_launchxl.rst

```diff
@@ -56,6 +56,12 @@ driver support.
 +-----------+------------+-----------------------+
 | GPIO      | on-chip    | gpio                  |
 +-----------+------------+-----------------------+
+| I2C       | on-chip    | i2c                   |
++-----------+------------+-----------------------+
+
+Note: For consistency with TI SimpleLink SDK and BoosterPack examples,
```

Comment:
If you want this note to stand out more, you could use this:

     .. note::
 
        For consistency with TI SimpleLink SDK and BoosterPack examples,
        the I2C driver defaults to I2C_BITRATE_FAST mode (400 kHz) bus
        speed on bootup.

Reply:
OK, added the .. note:: directive.